### ISSUE TITLE
[Linux] EncryptedLocalStore doesn't work

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#1829]
## Problem Description
`EncryptedLocalStore` doesn't work with Linux devices.
It just throw exception when you try to use it.
Tested with latest AIR 33.1.1.795 with Ubuntu 20.04.2.

It works fine with other platfroms (Windows, macOS, Android, iOS).
`EncryptedLocalStore.isSupported` always returns `true` in all these cases.

Related issue (that was earlier for Windows/macOS): https://github.com/airsdk/Adobe-Runtime-Support/issues/205

## Steps to Reproduce
Launch code below.

Application example with sources attached.
[encrypted_local_store_linux_bug.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/8442623/encrypted_local_store_linux_bug.zip)

```
var bytes:ByteArray = EncryptedLocalStore.getItem("test");
```

**Actual Result:**
Exception will be throwed:
```
Error: EncryptedLocalStore is out of memory
at flash.data::EncryptedLocalStore$/processErrorCode()
at flash.data::EncryptedLocalStore$/getItem()
```

**Expected Result:**
`EncryptedLocalStore.getItem` returns `ByteArray` with data of specified name without exceptions.

## Known Workarounds
none

**Comment by itlancer** [comment]
Issue still exists. With latest AIR 33.1.1.929 and Ubuntu 22.04.1 LTS actual result (error):
```
Error: EncryptedLocalStore database access error
	at flash.data::EncryptedLocalStore$/processErrorCode()
	at flash.data::EncryptedLocalStore$/getItem()
```

**Comment by itlancer** [comment]
@ajwfrost, issue still exists with latest AIR 50.0.0.1

**Comment by itlancer** [comment]
Fixed with latest AIR 50.0.1.1.
Thanks!
